app/models/download_agreement.rb

```diff
@@ -0,0 +1,34 @@
+class DownloadAgreement
```

Comment:
Put a comment at the top of this file outlining the general use case for this class

Reply:
2e76f7c